main.go

```diff
@@ -153,7 +185,7 @@ func main() {
 			)
 		}
 
-		baseURL, err := url.Parse(c.String("host") + "/api/v1/")
+		baseURL, err := url.Parse(c.String("host") + "/api/v1.1/")
```

Comment:
Did you mean to include this change? There is a PR over on `go-circleci` to switch or add support for the `v1.1` API, but just switching the path causes 404s for many endpoints as the `v1.1` API requires you to prefix the path with `/github` or `/bitbucket`